src/main/java/frc/robot/constants/FieldConstants.java

```diff
@@ -11,19 +11,10 @@ public class FieldConstants {
   // TODO: After testing, this value might need to be tuned. Change the 0+ to however much the robot needs to be away from the node tape
   public static final double kRobotDistanceFromNodeTape = 0 + DriveConstants.kRobotWidthWithBumpers / 2; // meters
 
-  // The locations of the node tape in the x axis, relative to the very bottom left corner of the entire field
-  public static final double kBlueNodeTapePosX = 4.379; // meters
-  public static final double kRedNodeTapePosX = 18.183; // meters
+  // The distance from the apriltag to the edge of the tape in front of the grids
+  public static final double kAprilTagOffset = Units.inchesToMeters(14.06 - 0.26); // meters
```

Comment:
So this is 14.25 inches (from the surface of the grid that the AprilTag is on to the end of the hybrid node divider) - 0.22 in (thickness of AprilTag plate) = 14.03 inches?